examples/docs/src/components/index.js

```diff
@@ -0,0 +1,10 @@
+export { default as theme } from "./theme";
+export { default as mdxProviderComponents } from "./mdxProviderComponents";
+export { default as ThemeProvider } from "./themeProvider";
+export { default as Code } from "./code";
+export { default as CodeBlock } from "./codeBlock";
+export { default as Container } from "./container";
+export { default as Heading } from "./heading";
+export { default as Notification } from "./notification";
+export { default as Pre } from "./pre";
+export { default as Text } from "./text";
```

Comment:
Is this different than `export Text from './text'`? Maybe this avoids having to import first?

Reply:
That's a good question, lemme try that out!

Reply:
Looks like we need to enable this syntax with a Babel plugin. Lemme create a follow-up issue / PR.

![image](https://user-images.githubusercontent.com/2344137/44950080-bb8c6a80-adf3-11e8-9e0b-91ae33c0621c.png)